src/08_comprehensions.py

```diff
@@ -12,13 +12,19 @@
 
 y = []
 
+for x in range(11):
+    y.append(x)
+
 print (y)
 
 # Write a list comprehension to produce the cubes of the numbers 0-9:
 # [0, 1, 8, 27, 64, 125, 216, 343, 512, 729]
 
 y = []
 
+for x in range(10):
```

Comment:
Same here, you can accomplish this easily in 1 line `y = [x**3 for x in range(10)]`